WrapImage/nifti_wrapper.py

```diff
@@ -58,7 +61,20 @@ def save_nifti_3d(data, output_file):
         bvals = read_json_file(bval_file)
 
         # Pass additional arguments to the algorithm
+        rng = np.random.RandomState(42)
+        fit = OsipiBase(algorithm=args.algorithm)
+        S0 = 1
+        gd = GenerateData(rng=rng)
```

Comment:
No need to generate data, that's just for testing.

Reply:
Hello, I do not quite understand, how do toget the signal value. Also, what are the typical algorithm that can be passed as argument. Also, where is the test file tested https://github.com/OSIPI/TF2.4_IVIM-MRI_CodeCollection/blob/main/tests/IVIMmodels/unit_tests/test_ivim_synthetic.py. Thank you.

Reply:
The signal comes from the loaded nifti data. The testing is entirely different and uses synthesized data. It's just an example of how to use the OsipiBase class.

The algorithms are listed [here](https://github.com/OSIPI/TF2.4_IVIM-MRI_CodeCollection/blob/main/doc/code_contributions_record.csv), but not all are wrapped yet.

The test file is run when you run `pytest`, but specifically the slow tests as is done in the [analysis](https://github.com/OSIPI/TF2.4_IVIM-MRI_CodeCollection/blob/main/.github/workflows/analysis.yml) workflow.